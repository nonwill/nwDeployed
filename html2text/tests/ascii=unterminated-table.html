<html>
<body>
<table>
<tr>
<td>Hello!</td>
</tr>
</body>
</html>
