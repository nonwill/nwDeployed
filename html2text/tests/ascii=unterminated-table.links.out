ascii=unterminated-table.default.out